core/src/main/resources/META-INF/resources/primefaces-extensions/scaffolding/scaffolding-widget.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * PrimeFaces Extensions Scaffolding Widget.
+ *
+ * @author Jasper de Vries jepsar@gmail.com
+ * @since 10.0.3
+ */
+PrimeFaces.widget.ExtScaffolding = PrimeFaces.widget.BaseWidget.extend({
+
+    /**
+     * Initializes the widget.
+     *
+     * @param {object}
+     *            cfg The widget configuration.
+     */
+    init: function (cfg) {
+        this._super(cfg);
+        this.id = cfg.id;
+        this.cfg = cfg;
+        this.async = cfg.async || true;
+        this.global = cfg.global || true;
+        this.loadWhenVisible = cfg.loadWhenVisible || false;
+        if (this.loadWhenVisible && !this.visible()) {
+            this.bindScrollMonitor();
+        } else {
+            this.loadData();
+        }
+    },
+
+    /**
+     * Trigger data loading via Ajax.
+     * @private
+     */
+    loadData: function () {
+        PrimeFaces.ab({
```

Comment:
Isn't PrimeFaces.ab the shorthand?  Should we be explicit here since it gets minified anyway?

Reply:
Ah, wasn't aware of that.. will change it